python-eval-mathrepl/mathrepl.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+# File: mathrepl.py
```

Comment:
Also probably don't need this line, since the file is named `mathrepl.py` :smile: